src/components/Containers/Accounts/AccountsList/AccountsListItem.js

```diff
@@ -51,7 +55,7 @@ const AccountsListItem = ({ account, setAccount, ...props }) => {
   const _deleteFile = account => {
     Axios.delete(DELETE_FILE, { data: account })
       .then(response => {
-        alert.success('The account has been restored.');
+        alert.success('The account has been restored or removed.');
```

Comment:
Ideally we'd warn the user before they take the action that "restoring" an account with only one file in it will remove the account. But that's maybe something for the future.

Reply:
I kind of tried to hint at that by changing the button text on a single file account to "Remove Account" rather than "Restore Account" but maybe that's too subtle. I can look into implementing the modal here though too.